gisMetadataProcessing.py

```diff
@@ -0,0 +1,81 @@
+from bs4 import BeautifulSoup
+import csv
+import os
+
+
+def creatkeyword(value):
+    """Create keyword element."""
+    keywordkey = keywordxml.new_tag(tag + 'key')
```

Comment:
This appears to be relying on `keywordxml` being defined at the global level. In general, global variables should be avoided. I'd definitely say in this case it should be passed into the function.